workers/ohsome_quality_analyst/api/api.py

```diff
@@ -116,13 +117,15 @@ async def post_indicator(
     p = parameters.dict()
     dataset = p["dataset"]
     fid_field = p.get("fid_field", None)
+    includeSvg = p["includeSvg"]
```

Comment:
Here I would rename the variable to adhere to naming conventions (snake case for variables). 
```suggestion
    include_svg = p["includeSvg"]
```

Reply:
Also you do not need to store it as a variable. You could pass it directly to the `_fetch_indicator` function.

```
 return await _fetch_indicator(
        name.value,
        p["layer_name"].value,
        p["includeSvg"],
        ...
```